tests/integration_tests/performance/configs/vsock_throughput_test_config.json

```diff
@@ -0,0 +1,478 @@
+{
+  "time": 20,
+  "server_startup_time": 2,
+  "load_factor": 1,
+  "modes": {
+    "g2h": [
+      ""
+    ],
+    "h2g": [
+      "-R"
+    ],
+    "bd": [
+      "",
+      "-R"
+    ]
+  },
+  "protocols": [
+    {
+      "name": "tcp",
+      "omit": 3,
+      "payload_length": [
+        "1024K",
+        "DEFAULT",
+        "1024"
+      ]
+    }
+  ],
+  "measurements": {
+    "throughput": "Mbps",
+    "duration": "seconds",
+    "cpu_utilization_vmm": "percentage",
+    "cpu_utilization_vcpus_total": "percentage"
+  },
+  "statistics": {
+    "throughput": [{
+      "name": "total",
+      "function": "Sum",
+      "criteria": "EqualWith"
+    }],
+    "duration": [{
+      "function": "Avg"
+    }],
+    "cpu_utilization_vmm": [{
```

Comment:
These should also have criteria

Reply:
Omitted this one, please take a look here: https://github.com/firecracker-microvm/firecracker/pull/2424/commits/97dbe9aef8c1c2301de0abfe43aab74897df55b9.